<launch>

  
  <param name="autobed/robot_description" textfile="$(find hrl_base_selection)/urdf/occupied_autobed.URDF" />
  <node name="autobed_state_publisher" pkg="robot_state_publisher" type="state_publisher" ns="autobed"/>
<node name="global_tf_broadcaster" pkg="hrl_base_selection" type="test_tf_broadcaster.py" output="screen"/>


</launch>
